runtime/oti/vmconstantpool.xml

```diff
@@ -412,7 +411,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<staticmethodref class="java/lang/J9VMInternals" name="recordInitializationFailure" signature="(Ljava/lang/Class;Ljava/lang/Throwable;)V"/>
 
 	<!-- Static method references needed to support OpenJDK MethodHandles. -->
-	<staticmethodref class="java/lang/invoke/MethodTypeHelper" name="vmResolveFromMethodDescriptorString" signature="(Ljava/lang/String;Ljava/lang/ClassLoader;Ljava/lang/Class;)Ljava/lang/invoke/MethodType;" flags="opt_openjdkMethodhandle"/>
+	<staticmethodref class="java/lang/invoke/MethodTypeHelper" name="vmResolveFromMethodDescriptorString" signature="(Ljava/lang/String;Ljava/lang/ClassLoader;Ljava/lang/Class;)Ljava/lang/invoke/MethodType;" flags="opt_methodHandleCommon"/>
```

Comment:
```suggestion
	<!-- Common method references shared between OpenJ9 and OpenJDK MethodHandles. -->
	<staticmethodref class="java/lang/invoke/MethodTypeHelper" name="vmResolveFromMethodDescriptorString" signature="(Ljava/lang/String;Ljava/lang/ClassLoader;Ljava/lang/Class;)Ljava/lang/invoke/MethodType;" flags="opt_methodHandleCommon"/>
	
	<!-- Static method references needed to support OpenJDK MethodHandles. -->
```

The comment needs to be shifted after the `opt_methodHandleCommon` method and a new comment added above

Reply:
fixed